src/Symfony/Component/Messenger/Command/ConsumeMessagesCommand.php

```diff
@@ -77,6 +77,7 @@ protected function configure(): void
                 new InputOption('queues', null, InputOption::VALUE_REQUIRED | InputOption::VALUE_IS_ARRAY, 'Limit receivers to only consume from the specified queues'),
                 new InputOption('no-reset', null, InputOption::VALUE_NONE, 'Do not reset container services after each message'),
                 new InputOption('all', null, InputOption::VALUE_NONE, 'Consume messages from all receivers'),
+                new InputOption('exclude-transports', 'et', InputOption::VALUE_REQUIRED | InputOption::VALUE_IS_ARRAY, 'Exclude specific receivers/transports from consumption (can only be used with --all)'),
```

Comment:
Following previous conversation, the option `--receivers` describes which **receivers/transports** to consume, but here `--exclude-transports` describes which **receivers/transports** to **not** consume.

I feel like we should either have `--receivers` and `--exclude-receivers`, or `--transports` and `--exclude-transports`, but not both mixed. 

I think the best option here is to go with `--exclude-receivers` (like your initial suggestion 😇).